tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +521,373 @@ def cvg_api(snappi_api_serv_ip,
+                                           port_type=SnappiPortType.VlanMember,
+                                           peer_port=phy_intf)
+
+            port_config_list.append(port_config)
+
+    return True
+
+
+def __intf_config_multidut(config, port_config_list, duthost, snappi_ports):
+    """
+    Configures interfaces of the DUT
+    Args:
+        config (obj): Snappi API config of the testbed
+        port_config_list (list): list of Snappi port configuration information
+        duthost (object): device under test
+        snappi_ports (list): list of Snappi port information
+    Returns:
+        True if we successfully configure the interfaces or False
+    """
+    prefix = 24
+    dutIps = create_ip_list("20.0.1.1", len(snappi_ports), mask=prefix)
+    tgenIps = create_ip_list("20.0.1.2", len(snappi_ports), mask=prefix)
+    ports = [port for port in snappi_ports if port['peer_device'] == duthost.hostname]
+
+    for port in ports:
+        port_id = port['port_id']
+        dutIp = dutIps[port_id]
+        tgenIp = tgenIps[port_id]
+        mac = __gen_mac(port_id)
+        logger.info('Configuring Dut: {} with port {} with IP {}/{}'.format(
+                                                                            duthost.hostname,
+                                                                            port['peer_port'],
+                                                                            dutIp,
+                                                                            prefix))
+        if port['asic_value'] == 'None':
+            duthost.command('sudo config interface ip add {} {}/{} \n' .format(
+                                                                                port['peer_port'],
+                                                                                dutIp,
+                                                                                prefix))
+        else:
+            duthost.command('sudo config interface -n {} ip add {} {}/{} \n' .format(
+                                                                                    port['asic_value'],
+                                                                                    port['peer_port'],
+                                                                                    dutIp,
+                                                                                    prefix))
+        device = config.devices.device(name='Device Port {}'.format(port_id))[-1]
+        ethernet = device.ethernets.add()
+        ethernet.name = 'Ethernet Port {}'.format(port_id)
+        ethernet.port_name = config.ports[port_id].name
+        ethernet.mac = mac
+        ip_stack = ethernet.ipv4_addresses.add()
+        ip_stack.name = 'Ipv4 Port {}'.format(port_id)
+        ip_stack.address = tgenIp
+        ip_stack.prefix = prefix
+        ip_stack.gateway = dutIp
+        port_config = SnappiPortConfig(
```

Comment:
Align function arguments.

Reply:
no we are not going to change this, as pre-commit checks have passed